src/admin-views/notices/virtual-events-sale.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * The Virtual Events Sale admin notice.
+ *
+ * @since TBD
+ *
+ * @var string $cta_url The short URL for black friday.
+ */
+?>
+<div class="tribe-marketing-notice-virtual_events_sale">
+    <div class="tribe-marketing-notice-virtual_events_sale-banner_ad-wrap">
+        <div class="tribe-marketing-notice-virtual_events_sale-banner_ad-title-wrap">
+            <div class="tribe-marketing-notice-virtual_events_sale-banner_ad-title">
+                <span class="tribe-marketing-notice-virtual_events_sale-banner_ad-red">S</span>
+                <span class="tribe-marketing-notice-virtual_events_sale-banner_ad-blue">A</span>
+                <span class="tribe-marketing-notice-virtual_events_sale-banner_ad-yellow">L</span>
+                <span class="tribe-marketing-notice-virtual_events_sale-banner_ad-green">E</span>
+            </div>
+        </div>
+        <div class="tribe-marketing-notice-virtual_events_sale-banner_ad-copy">
+            Get 20% off the Virtual Events Add-On with coupon code 
+            <span class="tribe-marketing-notice-virtual_events_sale-banner_ad-code">virtual-2021</span>
+            <span class="tribe-marketing-notice-virtual_events_sale-banner_ad-limit"> for a limited time only!</span>
+        </div>
+        <div class="tribe-marketing-notice-virtual_events_sale-banner_ad-button">
+            <a 
+                target="_blank" 
+                href="<?php echo esc_url( $cta_url ); ?>" 
+                class="tribe-marketing-notice-virtual_events_sale-banner_ad-link">
```

Comment:
whenever adding `target="_blank"`, you want to add a `rel="noopener noreferrer"` to the link.

```suggestion
            <a 
                target="_blank"
                rel="noopener noreferrer"
                href="<?php echo esc_url( $cta_url ); ?>"
                class="tribe-marketing-notice-virtual_events_sale-banner_ad-link"
            >
```